tests/http/stateless/ApplicationRestTest.php

```
@@ -37,6 +38,21 @@ public function testPrepareForRequestCallsOverriddenReinitializeApplicationHook(
         );
     }
 
+    /**
+     * @throws InvalidConfigException if the configuration is invalid or incomplete.
+     */
+    public function testPrepareForRequestCallsOverriddenResetRequestStateHook(): void
+    {
+        $app = $this->applicationRest();
+
+        $app->runPrepareForRequest(FactoryHelper::createRequest('GET', 'site/index'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii2\extensions\psrbridge\tests\support\FactoryHelper' in method 'testPrepareForRequestCallsOverriddenResetRequestStateHook'. (undefined)

(StaticAccess)